Svc/PassiveRateGroup/docs/PassiveRateGroup.md

```diff
@@ -0,0 +1,5 @@
+<title>PassiveRateGroup Component Dictionary</title>
+# PassiveRateGroup Component Dictionary
```

Comment:
These are artifacts of the old make system that were autogenerated. Were you planning to add the autogeneration back in?